workers/python/pyproject.toml

```diff
@@ -2,14 +2,16 @@
 package-mode = false
 
 [tool.poetry.dependencies]
-python = "3.11.10"
+python = "^3.11.9,<3.13"
```

Comment:
If I remember correctly, we must stick with 3.11, otherwise unstructured will not work. I'll retest to confirm.